packages/grafana-ui/src/components/ColorPicker/ColorPickerPopover.tsx

```diff
@@ -117,16 +118,18 @@ class UnThemedColorPickerPopover<T extends CustomPickersDescriptor> extends Reac
     const styles = getStyles(theme);
     return (
       <div className={styles.colorPickerPopover}>
-        <div className={styles.colorPickerPopoverTabs}>
-          <div className={this.getTabClassName('palette')} onClick={this.onTabChange('palette')}>
-            Colors
-          </div>
-          <div className={this.getTabClassName('spectrum')} onClick={this.onTabChange('spectrum')}>
-            Custom
+        <FocusScope contain restoreFocus autoFocus>
+          <div className={styles.colorPickerPopoverTabs}>
+            <div tabIndex={0} className={this.getTabClassName('palette')} onClick={this.onTabChange('palette')}>
```

Comment:
Considering using button here instead.

Reply:
Have you looked at this @thisisobate https://github.com/adobe/react-spectrum/blob/1e2b7f280a04ead83e21b27fa580eb189e5f4186/packages/%40react-aria/overlays/docs/DismissButton.mdx
We're using it here in the NavBar: https://github.com/grafana/grafana/blob/0611e435fc976e319ae71c9becbb5130feb6ec1d/public/app/core/components/NavBar/NavBarItemMenuTrigger.tsx#L140

Reply:
Haven't explored this approach yet. Would take it for a spin

Reply:
DismissButton doesn't seem to solve the `esc` issue in panelEdit.
I think my current solution already solves it.

Reply:
The DismissButton is for people that use screen readers and might not default to using the `esc` key.